code/modules/modular_computers/computers/subtypes/dev_wristbound.dm

```diff
@@ -22,26 +22,26 @@
 		else
```

Comment:
You don't need these `else`s now you stuck the `..()` on the end, is what I meant...